detectlanguage_test.go

```diff
@@ -0,0 +1,23 @@
+package cohere
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestDetectLanguage(t *testing.T) {
+	co, err := CreateClient(apiKey)
+	if err != nil {
+		t.Error(err)
+	}
+
+	t.Run("DetectLanguage success", func(t *testing.T) {
+		res, err := co.DetectLanguage(DetectLanguageOptions{
+			Texts: []string{"this text is in english", "Этот текст на Русском языке."},
+		})
+		assert.Nil(t, err)
```

Comment:
shouldn't we assert language name too? and maybe confidence (tricky since we don't mock of course, but at least that there exists a float > 0).